plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java

```diff
@@ -570,7 +571,7 @@ protected void resultSelect(Event e) {
 
             selectedValues.add(newValue);
 
-            if (!isMultiple() && allowSingleDeselect) {
+            if (!isMultiple() && isAllowSingleDeselect()) {
```

Comment:
@ewolk @okorokhina The change in this line introduced this bug with LabOrder -> ProviderName chosen => the missing X. See my comment https://github.com/lillegroup/escribehost/pull/1425#issuecomment-212586997